src/tree_data.h

```diff
@@ -163,7 +163,9 @@ struct lysc_type;
  * - ::lyd_new_list2()
  * - ::lyd_new_any()
  * - ::lyd_new_opaq()
+ * - ::lyd_new_opaq2()
```

Comment:
also the  transition manual needs an update

Reply:
Okay, it should be fixed now but I do not really get the added value of the changes I made.